README.md

```diff
@@ -23,38 +23,110 @@ Ansible Role for installing the Docker runtime environment on your (Debian) host
 
 ## Requirements
 
-*nothing special*
+This is intended for Debian GNU/Linux based systems.
+Plain old Debian works, Ubuntu should work, others might work.
 
 ## Install
 
-This role can be installed through your *requirements.yml*, either from
-[Ansible Galaxy](https://galaxy.ansible.com/) or through the Git repo.
+This role can be installed through your *requirements.yml*,
```

Comment:
Please only document the current status of the software. Future plans are part of an optional "roadmap" paragraph.